### PR TITLE
Matplotlib Serialization & PythreeJS Kaitlyn Chait

### PR DESCRIPTION
**Pull request by katierose1029**
Checklist for prospect students for Google Summer of Code 2017:

- [x] I contacted NumFOCUS mentors before writing my proposal
- [x] I showed you my contribution (it can be any form: proof-of-concept project idea, some sample code or just a link to your commits from other project)
- [x] I linked to my sample contribution from the proposal
- [ ] I linked to my opened issues in numfocus/gsoc repository from the proposal

### CODE REVIEW

**Comment by story645** [comment]
You didn't need to close your first one... And can you please change the title of this to your name and the project you're applying to?

**Comment by katierose1029** [comment]
I would like to hand this in but I want to make sure it is strong and ready to go.

**Comment by tacaswell** [comment]
attn @rmorshea

File: 2017/proposals/chait-kaitlyn.md [hunk 1]
```diff
@@ -0,0 +1,306 @@
...
+    How can we serialize a figure to json (or other format)?
+    When we have this ability, can we return the original matplotlib figure?
+
+**Phase 2:** (June 26th - July28th)
+    *The focus of this stage will be taking serialized data and rendering it
+    into a PythreeJS widget*
+    How can we take our serialized matplotlib data and render it to a
+    PythreeJS widget?
+
+**Phase 3:** (July 24th - August 25th)
```
Comment (tacaswell) [inline 2017/proposals/chait-kaitlyn.md]: This seems out of scope for matplotlib.

File: 2017/proposals/chait-kaitlyn.md [hunk 2]
```diff
@@ -0,0 +1,306 @@
...
+![PythreeJS_Sphere_Image](https://katierose1029.github.io/gsoc/PythreeJS_Sphere_Image.png)
+
+
+________________________________________________________________________________
+
+## Schedule of Deliverables
+
+**Community Bonding Period:** (May 1st - May 28th)
+
+**Phase 1:** (May 29th - June 23rd)
```
Comment (tacaswell) [inline 2017/proposals/chait-kaitlyn.md]: I think this is way underestimating the amount of time this will take.

Comment (katierose1029) [inline 2017/proposals/chait-kaitlyn.md]: I wasn't sure how to properly schedule this project. 
If I were to spend the first phase understanding the serialization of data 
and spend the second phase understanding how to implement it, would that more on target?

Comment (story645) [inline 2017/proposals/chait-kaitlyn.md]: Kinda - it's more of an agile/TDD approach, so more prototype serialization/iterate. But yes, it'd be more on target for serialization to take at least two phases and then push the rest of the project into "future work"

**Pull request merged**